packages/api-elements/lib/define-value-of.js

```diff
@@ -87,6 +87,14 @@ const isPrimitive = e => (e instanceof StringElement) || (e instanceof NumberEle
 const isEnumElement = e => e instanceof EnumElement;
 const isPlural = e => e instanceof ArrayElement;
```

Comment:
I'd name things based on what they do for clarity (perhaps in another PR)
```suggestion
const isArrayElement = e => e instanceof ArrayElement;
```